project-portfolio.php

```diff
@@ -15,7 +15,7 @@
 	<div class="content">
 		<?php
 		$selected_page = 'portfolio';
-		require 'includes/nav.php';
+		require 'includes/nav.php'
```

Comment:
Why have you taken this out? This semi-colon is something I said to put in in a previous comment.